### DISCUSSION TITLE
Logseq

### DISCUSSION THREAD

**Discussion by dhruvinsh** [catppuccin/catppuccin#1828] [Port Requests]
**Are you available to help in the creation of this port? 😄**
Maybe (As a Tester)

**Additional context 📷 (optional)**
https://logseq.com/ 
Logseq is a knowledge base like obsidian.md that works on top of local plain-text Markdown and Org-mode files. Use it to write, organize and share your thoughts, keep your to-do list, and build your own digital garden. Having a color scheme for the same is great idea.

**Comment by griimick** [comment]
I can package my custom theme for Logseq which is based on Catppuccin Mocha palette, although it is far away from Catppuccin's quality standards.

Commenting here for visibility and request patience :)

**Comment by ghost** [comment]
> I can package my custom theme for Logseq which is based on Catppuccin Mocha palette, although it is far away from Catppuccin's quality standards.
> 
> Commenting here for visibility and request patience :)

i'm also interested in catppuccin theme for logseq.
i think your custom theme can be a great starting point that can be tested and refined

**Comment by griimick** [comment]
I moved to Obsidian 95% and rarely use Logseq now. I still have the repo with wip commits, will tidy it up and get it out of my system this month.

**Reply by griimick** [reply]
Wrapping things from local repo here for release: https://github.com/griimick/logseq/pull/2

**Reply by griimick** [reply]
Most of the features have been themed and made a small demo release here https://github.com/griimick/logseq/releases/tag/v0.0.1. 

Will make changes to transfer over to the org and put it up on the logseq plugin marketplace.

**Comment by griimick** [comment]
https://github.com/griimick/logseq

Port ready for testing and org transfer. Getting it out was prioritised over covering all logseq features. So, feel free to open issues once the repo is available under Catppuccin org. 

I will publish the theme in the official logseq plugin marketplace after org transfer.

**Reply by griimick** [reply]
@sgoudham what are the next steps? Should I create a new issue to go forward with this?

**Reply by sgoudham** [reply]
Oooh I didn't realise you could make threads!

Sorry, we should have clarified that the staff team will transfer the discussion to an issue on the users behalf once it's all ready to review!

I'll do it soon once I'm back on my computer! @griimick 

Thanks for your work ♥️

**Reply by sgoudham** [reply]
Oh wait. Replies are threads ahaha. GitHub mobile moment!

**Reply by griimick** [reply]
Sure. Thanks!
